src/fidesops/schemas/dataset.py

```diff
@@ -79,6 +91,7 @@ class FidesopsDatasetField(DatasetField):
     """Extends fideslang DatasetField model with additional Fidesops annotations"""
 
     fidesops_meta: Optional[FidesopsMeta]
+    length: Optional[int]
```

Comment:
Would prefer we stash all these extensions in the "meta" object instead

Reply:
This has been moved